unstructured_inference/models/largemodel.py

```diff
@@ -0,0 +1,171 @@
+import os
+from typing import Optional
+
+import numpy as np
+import torch
+from PIL.Image import Image
+from transformers import (
+    AutoTokenizer,
+    DonutImageProcessor,
+    DonutProcessor,
+    VisionEncoderDecoderModel,
+)
+
+from unstructured_inference.inference.layoutelement import LocationlessLayoutElement
+from unstructured_inference.models.unstructuredmodel import UnstructuredElementExtractionModel
+
+MODEL_TYPES = {
+    "large_model": {
```

Comment:
can we call this something more specific, even `large_model_jun23` if not feeling especially inspired?
reason being, this will not be the last "large_model"

Reply:
I too don't think large_model is a good name.